pkg/linode-bs/controllerserver.go

```diff
@@ -21,6 +21,7 @@ import (
 const gigabyte = 1024 * 1024 * 1024
 const minProviderVolumeBytes = 10 * gigabyte
 const waitTimeout = 300
+const DevicePathKey = "devicePath"
```

Comment:
no need to export and maybe combine all these consts into one call while youre here?

Reply:
I'd prefer separate lines for each const for clarity / ease of reading.